src/components/settings-menu/DevSection.tsx

```diff
@@ -248,6 +248,12 @@ const DevSection = () => {
     setLoadingStates(prev => ({ ...prev, clearMmkvStorage: false }));
   };
 
+  const onPressNavigationEntryPoint = () =>
+    navigate(Routes.HARDWARE_WALLET_TX_NAVIGATOR, {
+      screen: Routes.PAIR_HARDWARE_WALLET_AGAIN_SHEET,
+      // params: { errorType: 'off_or_locked' },
```

Comment:
Comment left there, should it be there?

Reply:
i think it doesn't really matter, this callback is just used for testing in dev settings